starting-files/jvmrun

```diff
@@ -0,0 +1 @@
+-heapsize 25M -classpath /usr/extensions-1.14.3-full.jar -emain com.hms_networks.americas.sc.extensions.ExtensionsMain
```

Comment:
This doesn't really matter, but this is a pr for 1.15, so won't this get updated in a couple commits?

Reply:
Yes. To ensure consistency with release commits across all starter project-based code bases, I added this as 1.14.3. That way, when a release commit is merged for v1.15.0 in a future PR, the version number will still be updated in all expected locations.

Reply:
Would it make sense to have this be introduced as 
```
-heapsize 25M -classpath /usr/extensions-0.0.0-full.jar -emain com.hms_networks.americas.sc.extensions.ExtensionsMain
```
To make it clear that this is introduced as unusable until the next release?

Reply:
The `pom.xml` at this stage still contains `1.14.3` as the version number, so I think this is fine as-is. It is basically dropping in files that would already be expected here per our starter project structure/conventions.
- There is no technical reason that this would not function with v1.14.3, and having v0.0.0 would make the file more unusable since v0.0.0 doesn't exist.

Typically, we would expect this file to be present during the `RELEASE: X.Y.Z` commit stage of a release, so I introduced it with the version number that would be expected at this point in the Git history (given the pom.xml, etc.). This also allows us to change the value from `v1.14.3` to `v.1.15.0` as expected during the future release commit.

Reply:
Also, in other repositories, we have merged parts of a version to be released and maintained the existing values in all applicable locations, including the `jvmrun` file. The only difference here is that this file did not exist previously, because this repository was not entirely in sync with the upstream starter project.

Reply:
> That was the intent

If you and @it-hms both prefer to use `v0.0.0`, then I will make this change, but I think it makes the overall Git history messier because, at this point in the repository, one would expect this file to be at `v1.14.3`. 

`v0.0.0` seems arbitrary to me, is not a valid version number for this repository, and is not applicable to the current state of the repository.

Reply:
im fine with it as is. Im just offering an alternative.

Reply:
Ok. I'll leave it as-is.